docs/releases/1.11.txt

```diff
@@ -446,6 +446,14 @@ Protection against insecure redirects in :mod:`django.contrib.auth` and ``i18n``
 and :func:`~django.views.i18n.set_language` protect users from being redirected
 to non-HTTPS ``next`` URLs when the app is running over HTTPS.
 
+``get_or_create`` and ``update_or_create`` check keyword arguments and ``defaults``
+-----------------------------------------------------------------------------------
+
+:meth:`~django.db.models.query.QuerySet.get_or_create` and
+:meth:`~django.db.models.query.QuerySet.update_or_create` ensure that keyword
+arguments and defaults are model fields. Keyword arguments containing a double
+underscore (`__`) will not be checked.
```

Comment:
I think this is a separate issue. I'll look into it and open a new PR (if I can find a fix). I can open a ticket to track this issue as well if you want?